webapp/components/filtered_user_list.jsx

```diff
@@ -104,9 +104,26 @@ class FilteredUserList extends React.Component {
 
         let count;
         if (users.length === this.state.users.length) {
-            count = '(' + users.length + ')';
+            count = (
+                <FormattedMessage
+                    id='filtered_user_list.count'
+                    defaultMessage='{count} {count, plural, =0 {0 members} one {member} other {members}}'
+                    values={{
+                        count: users.length
+                    }}
+                />
+            );
         } else {
-            count = '(' + this.state.users.length + ')';
+            count = (
+                <FormattedMessage
+                    id='filtered_user_list.countTotal'
+                    defaultMessage='{count} {count, plural, =0 {0 members} one {member} other {members}} of {total} Total'
```

Comment:
Never mind. It must automatically treat count as a number since it works fine on pre-release